content/200-orm/100-prisma-schema/10-overview/04-location.mdx

```diff
@@ -65,28 +60,57 @@ my-app/
 ├─ ...
 ```
 
-### How to enable multi-file Prisma schema support
+### Usage
 
-Support for multiple Prisma Schema files is currently in preview. To enable the feature, add the `prismaSchemaFolder` feature flag to the `previewFeatures` field of the `generator` block in your Prisma Schema: 
+You can split your Prisma schema into multiple files by enabling the `prismaSchemaFolder` Preview feature on your `generator` block:
```

Comment:
```suggestion
You can split your Prisma schema into multiple files by enabling the `prismaSchemaFolder` preview feature on your `generator` block:
```
Or is the capital `P` some form of convention for preview features? 🤔

Reply:
> Or is the capital P some form of convention for preview features? 🤔

Yes indeed, we capitalize maturity stages: 
- Early Access
- Preview
- General Availability

See: https://www.prisma.io/docs/orm/more/releases